### PR TITLE
Digital Library - Avocada

### PR DESCRIPTION
**Pull request by pernillasterner**
## Netlify link
https://technigo-project-library.netlify.app/

## Collaborators
No collaborators this week

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+sortOptions.forEach((option) => {
+  option.addEventListener("click", (e) => {
+    handleSort(recipes, e);
+  });
+});
+
+// Event listner for search form
+searchForm.addEventListener("submit", (e) => handleSearchForm(recipes, e));
+
+// Load the initial list of dogs when the page loads
```
Comment (fridaforser) [inline js/main.js]: Just a little error I found, it says "dogs" instead of recipes :)

**Comment by fridaforser** [comment]
Hi,

Good job this week! You can definitely tell you are comfortable with writing a lot of code, so much that a newbie like myself had some trouble understanding some parts, but it was easy to follow the comments and the flow, and I trust that you tweaked the code and kept it as dry as you could. 

I however noticed that the sorting didn't really work, however I personally loved the checkboxes in the filtering section, so the user can make multiple choices - well done!

Overall, you did a great job and I really like the vibe of your library, I just wish I was able to understand more of the code and how it all works together, but that's on me for not having that much previous coding experience.

I only added one small comment with a little error I found :)

File: .DS_Store
Comment (NellaMori) [inline .DS_Store:1]: Please add .DS_Store to your .gitignore file (if you don't have one, you need to create it) and remove the file from the repo 👍

File: assets/.DS_Store
Comment (NellaMori) [inline assets/.DS_Store:1]: same here 🧹

**Review by NellaMori — CHANGES_REQUESTED** [review]
Hi Pernilla, well done finishing your Library app 📚 You are an high-achiever, your app looks fabulous and I appreciate the effort in trying to have a modular approach with this project. However, the filtering and sorting features are buggy and the behaviour doesn't work as expected. I strongly recommend to use a single script file and have all your functions there. Fix the buggy filters and sorting functionality and you're good to go :)

File: .DS_Store
Comment (pernillasterner) [reply .DS_Store:1]: Hi Antonella, the .DS_Store is already in my .gitignore file but perhaps I created it the initial commit. Thanks for letting me know :)

**Comment by pernillasterner** [comment]
@AntonellaMorittu @fridaforser 
Can you give me some more details about what is not working with the filter and sorting functionality so I have something to go on? :)
The filter dropdown works as long as you click on either clear all or close. Can´t find the bug when it comes to the sorting.

**Review by NellaMori — APPROVED** [review]

**Review by NellaMori — APPROVED** [review]

**Pull request closed without merging**